joomla/joomla-cms
# [5.1] HTML Mail template images

Pull Request for Issue ##40678 .

### Summary of Changes
When sending html mails created with com_mails it is possible to include images.

The function convertRelativeToAbsoluteUrls ensures that the images have absolute references so that they will be loaded in the email. However if the image has been added to the email using the media manager it will include loading="lazy" and data-path=

This pr strips any references to loading and data-path

https://www.caniemail.com/search/?s=loading

### Testing Instructions
Set the mail templates to use html
Add an image to one of the mail templates - i suggest the Global Configuration: Test Mail
Trigger the mail and check it

### Actual result BEFORE applying this Pull Request
The code for the image in the email will be
`<img src="https://j5.test/images/9.webp" width="1141" height="720" loading="lazy" data-path="local-images:/9.webp">
`

### Expected result AFTER applying this Pull Request
The code for the image in the email will be
`<img src="https://j5.test/images/9.webp" width="1141" height="720">
`


The originakl issue report was classified as a bug therefore it should be 5.0 but it seems more like a feature so I made it against 5.1 but happy to change that.


### Link to documentations
Please select:
